README.md

```diff
@@ -1,10 +1,48 @@
-## Step 0: Setting up your environment:
+### Step 0: Installing general tools 
+#### Installing Git
+Install Git by using this command:
+```bash
+apt-get install git
```

Comment:
We might need to give external links to install git. so that they can install git based on the os. This looks like ubuntu dependant. 
we can just give command to check if its installed like
`git --version` to make sure its installed

Reply:
Have changed this section, thank you.